bin/make_err

```diff
@@ -178,25 +178,25 @@ sub create_public ($) {
     print HEADER "\n/*********************/\n";
     print HEADER   "/* Major error codes */\n";
     print HEADER   "/*********************/\n\n";
-    foreach $name (keys %major) {
+    foreach $name (sort keys %major) {
         printf HEADER "#define %-20s (H5OPEN %s_g)\n",$name,$name;
     }
-    foreach $name (keys %major) {
+    foreach $name (sort keys %major) {
         printf HEADER "H5_DLLVAR hid_t %-20s /* %s */\n","${name}_g;",$major{$name};
     }
 
     # Iterate over all the minor error sections
     print HEADER "\n/*********************/\n";
     print HEADER   "/* Minor error codes */\n";
     print HEADER   "/*********************/\n";
-    while ( ($sect_name, $sect_desc) = each (%section)) {
-        print HEADER "\n/* $sect_desc */\n";
+    foreach $sect_name (sort keys %section) {
```

Comment:
Sort the errors alphabetically by name, so they have stable lists in the generated headers.